src/main/java/ch/njol/skript/expressions/ExprBrushableItem.java

```diff
@@ -0,0 +1,104 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.BrushableBlock;
+import org.bukkit.block.BlockState;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Dusted Item")
+@Description({
+	"Represents the item that is uncovered when dusting.",
+	"The only blocks that can currently be \"dusted\" are Suspicious Gravel and Suspicious Sand."
+})
+@Examples({
+	"send target block's brushable item",
+	"set {_gravel}'s brushable item to emerald"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.20+")
+public class ExprBrushableItem extends SimpleExpression<ItemStack> {
+
+	private static final boolean SUPPORTS_DUSTING = Skript.classExists("org.bukkit.block.BrushableBlock");
+
+	static {
+		if (SUPPORTS_DUSTING)
+			Skript.registerExpression(ExprBrushableItem.class, ItemStack.class, ExpressionType.SIMPLE,
+				"[the] %blocks% brush[able] item",
+				"%blocks%'[s] brush[able] item");
```

Comment:
It could easily just be me but when I read this syntax I feel like we're talking about the item to brush the block and not the resulting item, possibly a better name could be used?

`event-block's resulting item` is one option making this call generic enough other events with a similar use could be added in the future.

Secondly `event-block brushable item` doesn't make sense to exist, this is also a property of a block so SimplePropertyExpression should be usable here as well.